### PR TITLE
Release 2.8.1: Release branch

### PR DESCRIPTION
**Pull request by jnidzwetzki**
6906742c Adjust pgspot CI check
719c82d Allow manual index creation in CAggs
c8148237 Remove flaky test from bgw_db_scheduler
fee75d50 Fix join on time column of compressed chunk
e7269eef Fix segfault when INNER JOINing hypertables
ad72f321 Add missing ORDER BY clause to dist_move_chunk
37689623 Keep locks after reading job status
dcae749e Move perltidyrc to root
ba54a989 Fix error when waiting for remote COPY to finish
795c0610 Fix Continuous Aggregate migration policies
b9cd5836 Fix chunk exclusion for prepared statements and dst changes
6b947216 Fix continuous aggregate migrate check constraint
bbdf1d07 Change project management GH action
fcca2f19 Fix a deadlock in chunk decompression and SELECTs
f13f4cad Updated the text of the stalebot
f55fe082 Fix gapfill function signature
0b2edeb5 Fix an ereport syntax error when using PG 12.0
c507c614 Update the name of the automation token
0632d738 Fix CI coredump information print
688952e0 Update procedural.yml
81197f2b Remove support for procedures as custom checks
16ac5b1c error compressing wide table
30a2a2e1 Fix assertion failure in constify_now
7bd020ef Fix flaky pg_dump test
360e1283 Fix bgw_db_scheduler regresscheck configuration
90b23287 Include running query in CI coredump information
f52e975f Do not use row-by-row fetcher for parameterized plans
34eab319 Remove database port from test output
19de94cd Remove multiple PG configurations from regresscheck-t
9a8bc0e2 Improve SKIPS handling in regression tests
d9d80634 Fix pgspot update script check
c13a5421 Remove test case causing bgw_custom crash
ab4bb99c Add automation for waiting on author
59e8c861 Change parameter names of cagg_migrate procedure
ba1e7dac Add CAGG migration permission tests
6d3089a5 Fix planner chunk exclusion for VIEWs
e5add9b0 Fix chunk exclusion for space partitions in SELECT FOR UPDATE queries
fba4425b Make the copy_memory_usage test less flaky
bbcad882 Fix bad TupleTableSlot drop
05b3dfbe Fix a possible error while flushing the COPY data
69e64666 Fix changelog entries added in wrong place
99afe45c GROUP BY error when setting compress_segmentby with an enum column

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,31 @@
...
+* #4685 Improve chunk exclusion for space dimensions
+* #4696 Report warning when enabling compression on hypertable
+* #4720 Fix chunk exclusion for prepared statements and dst changes
+* #4737 Fix join on time column of compressed chunk
+* #4738 Fix the assorted epoll_ctl() errors that could occur with COPY into a distributed hypertable
+* #4739 Fix continuous aggregate migrate check constraint
+* #4745 Fix FK constraint violation error while insert into hypertable which references partitioned table
+* #4756 Improve compression job IO performance
+* #4760 Fix segfault when INNER JOINing hypertables
+* #4735 Allow manual index creation for CAggs
```
Comment (fabriziomello) [inline CHANGELOG.md:23]: Same as pointed out in the other release issue: https://github.com/timescale/timescaledb/pull/4773#discussion_r984809457

Comment (jnidzwetzki) [reply CHANGELOG.md]: @fabriziomello Thank you for the hint. It is fixed in the current version of the PR.

**Comment by akuzm** [comment]
> https://github.com/timescale/timescaledb/commit/bbdf1d079a67d9f3b23b64a97466c8b4d35ba440 Change project management GH action

Stuff like this is only meaningful for main branch, right? Stalebot too.

**Review by akuzm — APPROVED** [review]

**Comment by jnidzwetzki** [comment]
@akuzm Yes, this is only meaningful for the main branch. But I included these PRs/commits to make the list of unreleased commits shorter if we do a further patch version.

**Review by svenklemm — APPROVED** [review]

**Pull request merged**